### ISSUE TITLE
terminal appears to be unfocused when opening a new tab

### ISSUE THREAD

**Issue by cdepillabout** [cdepillabout/termonad#7]
When opening a new tab, it appears to be unfocused.  Here is a screen shot I took after pressing `Ctrl-Shift-t` to open up a new tab:

![file](https://user-images.githubusercontent.com/64804/44626802-0aeb1a80-a95e-11e8-8305-d11e19b19a49.png)

However, the tab actually does have focus.  If you press keys on the keyboard, you can type normally:

![file1](https://user-images.githubusercontent.com/64804/44626808-20604480-a95e-11e8-9cd8-0a4d84344db7.png)

If you switch to another window, or even just another tab and come back to the tab you just opened, it appears to have like normal.

I'm not sure why this is happening.  I don't know if it is a problem with Termonad's code, VTE, or GTK.  I'd really love if it was fixed though.  It is really annoying.

*edit*: Just to be clear, when I say that "it doesn't look like the terminal has focus", what I mean is that the cursor looks like the following: 

▯ 

You can see that the middle is "empty".  In a terminal that has focus, you can see that the cursor is "filled in":

▮

In the pictures above, you can see that even though the terminal has focus, the color in the middle of the cursor is "empty".  This is the bug.  If a terminal has focus, then the cursor should always be "filled in".

**Comment by Grendel-Grendel-Grendel** [comment]
I have been looking at this and I did not find the bug on my end which complicates things. My best guess is that it has to do with GTK. Though I doubt it, since you mentioned that changing windows causes things to behave like normal, it might have something to do with the window manager. As I am using a rather esoteric one (BSPWM) and doubt we share that. Again, I doubt that.

Another variable that appears to differ is our GTK themes. I'm using gtk-dark and if Iḿ not mistaken you are using a different one. Is it possible that there is a difference in _versions_ of GTK between us?

**Comment by cdepillabout** [comment]
@Grendel-Grendel-Grendel Thanks for looking into this.  As I stated above, I'm not sure where this bug is coming from, but if you're not seeing it, that's great news!  I guess that means that it is some problem with my local setup.  I'm guessing that the problem is with my local version of GTK or VTE, but you're right that it could also be caused by something like the window manager or GTK version.

However, I have tried this on two different versions of GTK/VTE (one on old Debian, and one on new NixOS), and they both showed this weird behavior.  Maybe that's an indication that it is not caused by GTK or VTE, but instead by my window manager (XMonad).

If another one or two people can confirm that this bug doesn't actually exist, it would be great to have that information as well.
